drape_frontend/animation/perspective_animation.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "drape_frontend/animation/base_interpolator.hpp"
+
+namespace df
+{
+
+class PerspectiveAnimation : public BaseInterpolator
+{
+  using TBase = BaseInterpolator;
+
+public:
+  static double GetRotateDuration(double startAngle, double endAngle);
+
+  PerspectiveAnimation(double duration, double startRotationAngle, double endRotationAngle);
```

Comment:
Constructors should be declared first (https://google.github.io/styleguide/cppguide.html#Declaration_Order)